mlir-tensorrt/compiler/test/Conversion/StablehloToTensorRT/stablehlo-to-tensorrt-trt10.mlir

```diff
@@ -241,15 +242,15 @@ func.func @scatter_slice_update_i1_axis1(%arg0: tensor<1x134xi1>, %arg1: tensor<
   return %0 : tensor<1x134xi1>
 }
 
-// CHECK-LABEL: @scatter_slice_update_i1
+// CHECK-LABEL: @scatter_slice_update_i1_axis1
 //       CHECK:     %[[v0:.+]] = tensorrt.slice %arg1[0, 1][1, 1][1, 1] : tensor<1x2xi32> to tensor<1x1xi32>
 //       CHECK:     %[[v1:.+]] = tensorrt.collapse_rank %[[v0]] : tensor<1x1xi32> to tensor<i32>
 //       CHECK:     %[[v2:.+]] = tensorrt.collapse_rank %arg2 : tensor<1x1x5xi1> to tensor<1x5xi1>
 //       CHECK:     %[[v3:.+]] = tensorrt.constant dense<1> : tensor<2xi32>
 //       CHECK:     %[[v4:.+]] = tensorrt.linspace[%[[v1]] : tensor<i32>] [ static] [%[[v3]] : tensor<2xi32>] : tensor<1x5xi32>
-//       CHECK:     %[[v5:.+]] = tensorrt.identity %arg0 : tensor<1x134xi1> to tensor<1x134xi32>
-//       CHECK:     %[[v6:.+]] = tensorrt.identity %[[v2]] : tensor<1x5xi1> to tensor<1x5xi32>
-//       CHECK:     %[[v7:.+]] = tensorrt.scatter_elements {axis = 1 : i64} data(%[[v5]] : tensor<1x134xi32>) indices(%[[v4]] : tensor<1x5xi32>) updates(%[[v6]] : tensor<1x5xi32>)
+//       CHECK:     %[[v5:.+]] = tensorrt.identity %[[v2]] : tensor<1x5xi1> to tensor<1x5xi32>
```

Comment:
For some reason, the instructions are swapped for me when I test locally.

Reply:
Will  need to change these to `CHECK-DAG`.

Anyway, it looks like these changes will cause failure in CI. I suspect you are somehow on an old LLVM commit. Can you drop the change, push, and let's see if CI passes?